tests/Controller/UserControllerTest.php

```diff
@@ -246,35 +254,64 @@ public function testEditActionBeyondMaxFields()
 
         $formValues = $form->getValues();
 
-        $response = new Response();
+        $this->assertResponseIsSuccessful();
 
-        $this->assertEquals(Response::HTTP_OK, $response->getStatusCode());
+        $this->assertStringNotContainsString('L&#039;utilisateur a bien été modifié.', $this->client->getResponse()->getContent());
 
-        $this->assertNotContains('L&#039;utilisateur a bien été modifié.', $this->client->getResponse()->getContent());
+        $user = $this->entityManager->getRepository(User::class)->find(1);
 
-        $user = $this->manager->getRepository('AppBundle:User')->find(1);
-
-        $passwordVerification = $this->client->getContainer()->get('security.password_encoder')->isPasswordValid($user, $formValues['user[password][first]'], $user->getSalt());
+        $passwordVerification = $this->client->getContainer()->get(UserPasswordHasherInterface::class)->isPasswordValid($user, $formValues['user[password][first]']);
 
         $this->assertNotEquals($formValues['user[username]'], $user->getUsername());
         $this->assertFalse($passwordVerification);
         $this->assertNotEquals($formValues['user[email]'], $user->getEmail());
     }
 
+    public function testDeleteUserAction()
+    {
+        $userData = [
+            'username' => 'User2',
+            'password' => 'motdepasse',
+            'email' => 'user2@orange.fr'
+        ];
+        $user = new User();
+        $user->setUsername($userData['username']);
+        $user->setPassword($userData['password']);
+        $user->setEmail($userData['email']);
+        $this->entityManager->persist($user);
+        $this->entityManager->flush();
+
+        $this->client->loginUser($this->user);
+
+        $urlGenerator = $this->client->getContainer()->get('router');
+
+        $this->client->request(Request::METHOD_GET, $urlGenerator->generate('app_user_delete', ['id' => 2]));
```

Comment:
à tester si tu peux remplacer 2 par $user->getId()

Reply:
Fixed => See commit [Issue #4 Implementation of automated tests](https://github.com/l-lemaitre/todolist/pull/9/commits/cddb13e1ca46e6ffbaba7587e68c926c7aedbaa7)